library/x509write_csr.c

```diff
@@ -122,67 +130,66 @@ int mbedtls_x509write_csr_set_ns_cert_type( mbedtls_x509write_csr *ctx,
     return( 0 );
 }
 
-int mbedtls_x509write_csr_der( mbedtls_x509write_csr *ctx, unsigned char *buf, size_t size,
-                       int (*f_rng)(void *, unsigned char *, size_t),
-                       void *p_rng )
+static int mbedtls_x509write_csr_der_internal( mbedtls_x509write_csr *ctx, unsigned char *buf,
```

Comment:
Please avoid lines longer than 80 characters and indent uniformly. While there are other places in the library where we violate this, we should make sure we're eliminating them as much as possible, and not introduce any more of them.